.github/workflows/go-check.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         uses: protocol/.github/.github/actions/read-config@master
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.20.x
+          go-version: 1.21.x
```

Comment:
```suggestion
          go-version: "1.21.x"
```

Reply:
Nitpick just to avoid anyone using `1.30` unquoted in the future... as that gets parsed as a float and installs go1.3

Reply:
Help, yaml. Thanks